ESP/I2C_ESP32/main/WIFI.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef WIFI_HPP
+#define WIFI_HPP
+
+// #include "I2C_ESP.hpp"
+#include <iostream>
+
+/**
+ * @file      WIFI.hpp
+ * @brief     This class can preform data communication through wifi.
+ * @author    Kiet van Osnabrugge
+ * @license   MIT
+ */
+class WIFI {
```

Comment:
Class name and filename are incorrect. Filename should be: wifi.hpp Class name should be: WiFi (Wireless Fidelity)